src/components/prompts/ReactRouterConfirmLeave.js

```diff
@@ -2,26 +2,26 @@ import React from 'react';
 import { Prompt } from 'react-router-dom';
 
 /**
- * Customize React Router on leave prompt to use <OnLeavePrompt>
+ * Customize React Router on leave/unmount prompt to use <OnLeavePrompt>
  *
  * Hijacks history to show a modal first before
  * letting history take its course.
  * 
  * @param props {object}
- * @param props.prompt {function}
+ * @param props.askForFeedback {function}
  * @param props.message {string} - Passed indirectly to <OnLeavePrompt>
  * @param props.confirmer {object} - For hijacking history
- * @param props.isBlocking {boolean} - Whether the prompt should be shown
+ * @param props.isBlocking {boolean} - Whether this prompt should be shown
```

Comment:
This component doesn't return any visible component. It just sets up a callback. But maybe it is calling the component a prompt in the first place which is the problem. Or maybe I'm just being too picky.

Reply:
Yeah, I'm still not sure about the definition of prompt. Maybe it's a word we should move away from. Though the folder is called 'prompts'...

Would this work better? 'Whether to interrupt the user's navigation.'